tutorials/vpn-interop-guides/interop-guide-template/index.md

```diff
@@ -0,0 +1,1502 @@
+# Google Cloud VPN Interop Guide
+
+Using Cloud VPN with <vendor-name><product-name>Learn how to build site-to-site IPsec VPNs between [Cloud VPN
+](https://cloud.google.com/vpn/docs/)on Google Cloud Platform (GCP) and
+<vendor-name><product-name>.
+
+<To see a finished version of this guide, see the
+[Using Cloud VPN with Cisco ASR](https://cloud.google.com/community/tutorials/using-cloud-vpn-with-cisco-asr#top_of_page).>
+
```

Comment:
Somewhere should add a note telling the author to put their copy of this file in the /tutorials directory.